app/views/budgets/index.html.erb

```diff
@@ -6,22 +6,53 @@
 
 <% content_for :sidebar do %>
   <div flex layout-padding>
-    <br><br>
-    <%= render 'form' %>
+    <h3>Create Bugget</h3>
+
+<%= form_for(@budget) do |f| %>
```

Comment:
Is there a reason to move this from the form partial to the index file?

I think we should keep the same organization across the different views, so would be better to keep the forms partial file.

Reply:
That's a good point for consistency.
I think let's put this feedback as resolved and I will create new branch for formatting all the form files.